app/assets/stylesheets/single-post-view.scss

```diff
@@ -22,6 +21,9 @@
         padding-left: 10px;
       }
     }
+
+    .author-name { color: $grey; }
+
```

Comment:
This change is needed because `.author-name` no longer inherits the color of `.author`.